### DISCUSSION TITLE
What does it mean to say that only a single shadow map is supported?

### DISCUSSION THREAD

**Discussion by jwwalker** [google/filament#3812] [Q&A] [answered]
The documentation of `Renderer::render` remarks that "currently only a single shadow map is supported".  At first I thought that meant that only a single light can cast shadows, but that's apparently not true:  In `material_sandbox`, both the directional light and the spot light can cast shadows.  So what does it mean, in terms relevant to users of Filament, not developers of Filament?

**Comment by romainguy** [comment, marked as answer]
Only one directional shadow map is supported. Multiple spot shadows are supported.

@bejado We should update the docs.

**Reply by jwwalker** [reply]
When you say "multiple spot shadows", does that include point lights too?

**Reply by romainguy** [reply]
Point light shadows are currently not handled.
